src/endpoint.cpp

```diff
@@ -44,9 +44,22 @@ namespace oxen::quic
     {
         for (const auto& c : conns)
             c.second->io_trigger->on<uvw::AsyncEvent>(async_cb);
+    }
 
-        // for (const auto& c : conns)
-        //     c.second->io_ready();
+    void Endpoint::get_active()
+    {
+        for (const auto& c : conns)
+            log::info(log_cat, "Conn ID: {}, Remote Address: {}", c.first, c.second->remote);
+    }
+
+    std::list<std::pair<ConnectionID, Address>> Endpoint::get_conns()
```

Comment:
feels weird that `get_conn` below returns a `Connection*` but this returns a collection of `ConnectionID` (and `Address`), despite the function names being the same but plural.